### PR TITLE
Add informative reference to JSONPath draft

### PR DESCRIPTION
**Pull request by timbray**

### CODE REVIEW

File: draft-ietf-jsonpath-iregexp.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ normative:
   RFC7405: abnf-cs
 
 informative:
+  JSONPATH:
```
Comment (cabo) [inline draft-ietf-jsonpath-iregexp.md]: We generally let bib.ietf.org do the work, reducing this to:

```
informative:
  I-D.ietf-jsonpath-base: -jsonpath
```

File: draft-ietf-jsonpath-iregexp.md [hunk 2]
```diff
@@ -77,6 +81,10 @@ I-Regexp is a subset of XSD regular expressions {{XSD-2}}.
 
 This document includes guidance for converting I-Regexps for use with several well-known regular expression idioms.
 
+The development of I-Regexp was motivated by the work of the JSONPath Working Group. The Working Group wanted to include
+in its draft{{JSONPATH}} support for the use of regular expressions in JSONPath filters, but was unable to find a useful
```
Comment (cabo) [inline draft-ietf-jsonpath-iregexp.md]:
```suggestion
in its specification {{-jsonpath}} support for the use of regular expressions in JSONPath filters, but was unable to find a useful
```

**Review by cabo — APPROVED** [review]

File: draft-ietf-jsonpath-iregexp.md (hunk 1 above)
Comment (timbray) [reply draft-ietf-jsonpath-iregexp.md]: OK

File: draft-ietf-jsonpath-iregexp.md (hunk 2 above)
Comment (timbray) [reply draft-ietf-jsonpath-iregexp.md]: OK

File: draft-ietf-jsonpath-iregexp.md (hunk 1 above)
Comment (timbray) [reply draft-ietf-jsonpath-iregexp.md]: Doesn't seem to work. I get this:

Draft now has:
```
informative:
  I-D.draft-ietf-jsonpath-base: -jsonpath
  RE2:
(continues)
```

```
cat draft-ietf-jsonpath-iregexp.md  | kramdown-rfc2629 --v3 | lib/add-note.py | xml2rfc -q -s 'Setting consensus="true" for IETF STD document' --rfc-base-url https://www.rfc-editor.org/rfc/ --id-base-url https://datatracker.ietf.org/doc/html/ --cache=/Users/timothybray/.cache/xml2rfc --v2v3 /dev/stdin -o /dev/stdout >draft-ietf-jsonpath-iregexp.xml
*** no alias replacement for {{-jsonpath}}
/Users/timothybray/.cache/xml2rfc/reference.I-D.draft-ietf-jsonpath-base.xml: fetching from https://datatracker.ietf.org/doc/bibxml3/draft-ietf-jsonpath-base.xml
xml2rfc -q -s 'Setting consensus="true" for IETF STD document' --rfc-base-url https://www.rfc-editor.org/rfc/ --id-base-url https://datatracker.ietf.org/doc/html/ --cache=/Users/timothybray/.cache/xml2rfc draft-ietf-jsonpath-iregexp.xml -o draft-ietf-jsonpath-iregexp.txt --text --no-pagination
draft-ietf-jsonpath-iregexp.xml(0): Error: IDREF attribute target references an unknown ID "_jsonpath", at None
/Users/timothybray/dev/iregexp/draft-ietf-jsonpath-iregexp.xml(10): Error: Invalid document before running preptool.
Unable to complete processing draft-ietf-jsonpath-iregexp.xml
make: *** [draft-ietf-jsonpath-iregexp.txt] Error 1
rm draft-ietf-jsonpath-iregexp.xml
```

Comment (cabo) [reply draft-ietf-jsonpath-iregexp.md]: Sorry, this needs to be:

```
informative: 
  I-D.draft-ietf-jsonpath-base:   jsonpath
```

(The `-` is at the reference, not here.)
Apparently, I typed this from an airport...

**Pull request merged**